geodesic_library/hedgehog_mesh.txt

```diff
@@ -0,0 +1,897 @@
+300 596

```

Comment:
This seems like an example, and I suggest to put it under `data` folder

Reply:
Will do, thanks! But I added these code directly from the Google Code repo, i.e., https://code.google.com/archive/p/geodesic/